src/hotspot/share/opto/live.cpp

```diff
@@ -88,6 +87,9 @@ void PhaseLive::compute(uint maxlrg) {
 
   _free_IndexSet = nullptr;
 
+  Block_List wl;
+  _worklist = &wl;
+
```

Comment:
Now `_worklist` is a dangling pointer to released stack memory at the end of this method.  How do we make sure it isn't used?

Reply:
I removed the `_worklist` local variable and pass it into the functions that need it instead.